lcls-twincat-common-components/Library/Version/Global_Version.TcGVL

```diff
@@ -7,7 +7,7 @@
 // This function has been automatically generated from the project information.
 VAR_GLOBAL CONSTANT
     {attribute 'const_non_replaced'}
-    stLibVersion_lcls_twincat_common_components : ST_LibVersion := (iMajor := 0, iMinor := 0, iBuild := 0, iRevision := 0, nFlags := 1, sVersion := '0.0.0');
+    stLibVersion_lcls_twincat_common_components : ST_LibVersion := (iMajor := 0, iMinor := 0, iBuild := 0, iRevision := 0, nFlags := 0, sVersion := '0.0.0');
```

Comment:
this is weird, 0 is the correct flag here so this commit fixes it, but how did it get set to 1? (1 = released version that cannot be edited)

Reply:
I did a -nrw version of this library, saved and installed it. I think that set the nFlags to 1. I then changed the library name back to normal without the -nrw, and in order for the Global_Version.TcGVL to auto generate, I saved the library again, without install. This seemed to work? I must be doing something weird though because this file gets changed almost every time I work on this repo.

Reply:
I think what I expect is for this file to change any time the library name, the version, or the "released" checkbox changes

Reply:
And I think the "normal" version is the all zeroes.

We can also nuke this file altogether in the options if we want to, tc-release will rebuild it anyway during a tag.